src/GitReleaseNotes/ReleaseNotesGenerator.cs

```diff
@@ -27,19 +32,11 @@ public SemanticReleaseNotes GenerateReleaseNotes()
         {
             var context = _context;
 
-            using (var gitRepoContext = GetRepository(context, _fileSystem))
+            using (var temporaryFilesContext = new TemporaryFilesContext())
```

Comment:
I think the default for this command line tool should be to generate the file in the repository, not in a temp file. I found that change really annoying